AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -23,11 +24,11 @@ public class AccountController : Controller
     {
         private readonly UserManager<ApplicationUser> _userManager;
         private readonly SignInManager<ApplicationUser> _signInManager;
-        private readonly IOptions<GeneralSettings> _generalSettings;
         private readonly IMediator _mediator;
         private readonly IExternalUserInformationProviderFactory _externalUserInformationProviderFactory;
         private readonly IRedirectAccountControllerRequests _redirectAccountControllerRequests;
         private readonly string _externalCookieScheme;
+        private readonly TimeZoneInfo _timeZone = TimeZoneInfo.Local;

```

Comment:
No need to apologize. The DateTime and TimeZone code in AllReady is confusing and that is not your fault.